pkg/api/api.go

```diff
@@ -188,3 +188,51 @@ func UpdateLicense(c *gin.Context) {
 	c.JSON(http.StatusOK, res)
 
 }
+
+func SearchInLicense(c *gin.Context) {
+	field := c.Query("field")
+	search_term := c.Query("search_term")
```

Comment:
I would suggest to read the search term in the body. Keeping it in query limits the size of text someone can search.

Can we discuss it tomorrow @k-avy @avinal @ag4ums ?